README.md

```diff
@@ -37,7 +37,13 @@ This project depends on many third-party libraries and tools, credits to them fo
 * **Generate the proper app configuration using the `generate_emu_config` tool.**  
 * **If things don't work, try the `ColdClientLoader` setup.**  
 
-You can find helper guides, scripts, and tools here in this wiki: https://github.com/otavepto/gbe_fork/wiki/Emu-helpers  
+You can find helper guides, scripts, and tools here:
```

Comment:
should indicate these are 3rd party sources i think

Reply:
How is it now ?